controller.py

```diff
@@ -93,6 +93,8 @@ def __init__(self, app, local_path='', progress_bar=None):
             'language': {
                 'cs_CZ': 'Czech',
                 'en_US': 'English'
+				'de_US': 'Deutsch'
```

Comment:
There are also semicolons missing there, this gives syntax errors. I'll prepare a cleaned up version of the pull request.